src/components/navbar.tsx

```diff
@@ -13,8 +18,16 @@ export const Navbar = () => {
         </div>
         <h1>BeaverHacks Official Judging Platform</h1>
       </Link>
-      
-      <Link href='/Login' className='flex items-center px-5 hover:bg-gray-200'>Login</Link>
+      {session?.user ? (
+        <Button onClick={async () => {await signOut();}}>
+          Sign Out
+        </Button>
+      ) : (
+        <>
+          <Link href='/Login' className='flex items-center px-5 hover:bg-gray-200'>Login</Link>
+          <Link href='/SignUp' className='flex items-center px-5 hover:bg-gray-200'>Sign Up</Link>
```

Comment:
Wrap these in a div or smth so that they're properly on the righthand side. also please use the button component with the links so they look a bit nicer